draft-ietf-ppm-dap.md

```diff
@@ -2436,195 +2436,55 @@ suite:
+   aggregate share requests.
+1. Collection requests may leak information beyond the collection results. For
+   example, the Poplar1 VDAF {{!VDAF}} can be used to compute the set of heavy
+   hitters among a set of arbitrary bit strings uploaded by Clients. This
+   requires multiple evaluations of the VDAF, the results of which reveal
+   information to the Aggregators and Collector beyond what follows from the
+   heavy hitters themselves. Or the result of the Prio3Sum VDAF could leak
+   information about outlier values. This leakage can be mitigated using
+   differential privacy ({{dp}}).
 
 [[OPEN ISSUE: Should we have authentication in either direction between the
 Helper and the Collector? #155]]
 
-### Aggregator collusion
-
-If all Aggregators collude (e.g. by promiscuously sharing unencrypted input
-shares), then none of the properties of the system hold. Accordingly, such
-scenarios are outside of the threat model.
-
-### Attacker on the network {#network-attacker}
-
-We assume the existence of attackers on the network links between participants.
-Most passive network attacks are mitigated by DAP's requirement of HTTPS for all
-traffic and mutual authentication for key protocol interactions (see
-{{message-transport}}). Nonetheless, there remain information leaks that
-deployments should be aware of.
-
-#### Capabilities
-
-1. Attackers may observe messages exchanged between participants at the IP
-   layer.
-   1. The attacker can observe source and destination IP addresses, potentially
-      revealing the existence of Clients and Aggregators.
-   1. The time of upload of reports by Clients could reveal information about
-      user activity. For example, if a user opts into a new feature, and the
-      Client immediately reports this to Aggregators, then just by observing
-      network traffic, the attacker can infer what the user did.
-   1. Observation of message size could allow the attacker to learn how many
-      reports are being uploaded by a Client. For example, if the attacker
-      observes an encrypted message of some size, they can infer the size of the
-      plaintext, plus or minus the cipher block size. From this they may be able
-      to infer which VDAF is in use and perhaps which task the Client is
-      uploading reports for.
-   * These attacks can be mitigated by requiring Clients to submit reports at
-     regular intervals and independently of whether the event that the task is
-     tracking has not occurred, so that the absence of reports cannot be
-     distinguished from their presence.
-1. Tampering with network traffic. Attackers may drop messages or inject new
-   messages into communications between participants.
-   * DAP mitigates this by using standard HTTP semantics to allow requests to be
-     retried. However attacks that completely deny network access to
-     participants are outside of DAP's scope.
-
 [[OPEN ISSUE: The threat model for Prio --- as it's described in the original
```

Comment:
This open issue is redundant and can be removed. We also plan to call out this threat model in the VDAF draft: https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/304